flytekit/core/base_sql_task.py

```diff
@@ -24,9 +25,9 @@ def __init__(
         query_template: str,
         task_config: Optional[T] = None,
         task_type="sql_task",
-        inputs: Optional[Dict[str, Tuple[Type, Any]]] = None,
```

Comment:
> Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.

https://docs.python.org/3.7/library/stdtypes.html#dict

flytekit only supports 3.8+, do we still need to use OrderedDict here?

Reply:
https://mail.python.org/pipermail/python-dev/2017-December/151283.html

Reply:
I know that the dictionary after Python 3.7 is ordered. I use `OrderedDict` because I found somewhere in the codebase still uses `OrderedDict`. After some discussions in meetings, it seems that those are legacy codes. Therefore, I've rolled back the changes of `OrderedDict`.